### DISCUSSION TITLE
Umfrage zum Thema "Akzeptanz und Nutzung der elektronischen Identität (e-ID) in der Schweiz"

### DISCUSSION THREAD

**Discussion by SimonRofalski** [e-id-admin/open-source-community#26] [General]
### **Teilnehmende für meine Bachelorarbeit gesucht! 🚀**

Im Rahmen meiner Bachelorarbeit zum Thema "Akzeptanz und Nutzung der elektronischen Identität (e-ID) in der Schweiz" führe ich aktuell eine quantitative Umfrage durch. Ich würde mich freuen, wenn Du einen kurzen Moment Deiner Zeit opfern könntest, um an meiner Studie teilzunehmen.

👉 [https://bfh.fra1.qualtrics.com/jfe/form/SV_5795BWidkamYAcu](https://bfh.fra1.qualtrics.com/jfe/form/SV_5795BWidkamYAcu)
![SV_5795BWidkamYAcu-qrcode (1)](https://github.com/e-id-admin/open-source-community/assets/101046616/95bcdcab-5b01-4358-a36e-dee79153625b)

Deine Meinung ist wertvoll und trägt dazu bei, ein umfassendes Bild zum Verständnis der e-ID-Nutzung in der Schweiz zu zeichnen.
Ich danke Dir im Voraus für Deine Unterstützung und Deine wertvolle Zeit!

**Comment by stefan2904** [comment]
@SimonRofalski wäre es möglich, dass du die Ergebnisse bzw. deine Bachelorarbeit dann hier postest? Danke!
